### PR TITLE
docs(pattern): Login pattern from IoT

### PR DESCRIPTION
**Pull request by connor-leech**
This is a contribution from IoT.

Original content: https://pages.github.ibm.com/Watson-IoT/IoT-Design-Site/patterns/login-page/overview/

### CODE REVIEW

File: src/pages/experimental/login-pattern/index.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: Login page
```
Comment (janchild) [inline src/pages/experimental/login-pattern/index.mdx]:
```suggestion
title: Login pattern
```

File: src/pages/experimental/login-pattern/index.mdx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+[Experimental](/experimental/overview)
+
+#### Maintainer:
+
+[Chris Reckling](https://github.com/creckling)
+
+<PageDescription>
+
+The login pattern lets the user authenticate with the system so that they can have authorized access to product functionality.
```
Comment (janchild) [inline src/pages/experimental/login-pattern/index.mdx]:
```suggestion
The login pattern lets the user authenticate with a system for authorized access to product functionality.
```

File: src/pages/experimental/login-pattern/index.mdx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+</PageDescription>
+
+## Overview
+
+Login patterns ensure a user can authenticate into the system so they can access their data securely. Mobile login screens should follow the same pattern and illustration style as the desktop experience.
```
Comment (janchild) [inline src/pages/experimental/login-pattern/index.mdx]:
```suggestion
Login patterns ensure a user can authenticate into a system so they can access their data securely. Mobile login screens should follow the same pattern and illustration style as the desktop experience.
```

File: src/pages/experimental/login-pattern/index.mdx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+The "Forgot Password" pattern is separate from the authentication and login pattern. If there are several products being a login, a user should be able to choose which product log into.
```
Comment (janchild) [inline src/pages/experimental/login-pattern/index.mdx]:
```suggestion
The "Forgot Password" pattern is separate from the authentication and login pattern. If there are several products requiring a login, a user should be able to choose which product to log into.
```

File: src/pages/experimental/login-pattern/index.mdx [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+<Column colLg={8}>
+
+![Example of a login screen with a simple illustration](images/login-pattern-example2.png)
+
+</Column>
+</Row>
+
+### Login form on the right
+
+Right-side logins are ideal for editorial experiences, and instances where there is visual content featured on the left side of the window.
```
Comment (janchild) [inline src/pages/experimental/login-pattern/index.mdx]:
```suggestion
Right-side logins are ideal for editorial experiences and instances where there is visual content featured on the left side of the window.
```

**Comment by connor-leech** [comment]
@alisonjoseph I agree! Just bumped the image sizes up, merging now 🎉

**Pull request merged**